hw2/src/main/java/com/example/hw2/service/CityService.java

```diff
@@ -24,10 +27,17 @@
 	private WeatherService weatherService;
 	
 	public CityInfo getCityInfo(String cityName) {
-		
-		// TODO your code goes here
-		// delete the following line
-		return null; 
+		List<City> cities = cityRepository.findByName(cityName);
+		if (cities.size() < 1) {
+			return null;
+		}
+
+		City city = cities.get(0);
+		Country country = countryRepository.findByCode(city.getCountryCode());
+		TempAndTime tempAndTime = weatherService.getTempAndTime(cityName);
+		String time = Long.toString(tempAndTime.time);
+
+		return new CityInfo(city, country.getName(), tempAndTime.temp, time);

```

Comment:
Add a method to get the temp
```suggestion
		return new CityInfo(city, country.getName(), tempAndTime.getTemp(), time);

```

Reply:
Implemented, thanks